File: Segundo-Bimestre/Parcial 1/Arreglos/Arreglos_python/VectorPorEscalar.py
print("")
# Inicializar las variables
i = 0; n = 0; escalar = 0

# Ingreso de datos
n = int(input("Ingrese el tamaño del vector: "))

# Declarar los vectores
A = [None]*n
B = [None]*n

escalar = int(input("Ingrese el escalar: "))

# Ingresar los elementos del vector
for i in range (0, n):
    A[i] = int(input("Ingrese el elemento A["+str(i)+"]: "))

# Multiplicar por el escalar
for i in range (0, n):
    B[i] = A[i]*escalar

# Presentar el vector B
for i in range (0, n):
    print("B ["+str(i)+"] = ",B[i])

File: Segundo-Bimestre/Parcial 1/Arreglos/Arreglos_java/src/arreglos/VectorPorEscalar.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package arreglos;
import java.util.Scanner;
/**
 *
 * @author DELL
 */
public class VectorPorEscalar {
    public static void main(String[] args) {
        Scanner leer = new Scanner (System.in);
        // Declarar e inicializar las variables
        int i, n, escalar;
        i = 0; n = 0; escalar = 0;
        
        // Ingreso de datos
        System.out.print("Ingrese el tamaño del vector: ");
        n = leer.nextInt();
        
        // Declarar vectores
        int [] A = new int [n];
        int [] B = new int [n];
        
        System.out.print("Ingrese el escalar: ");
        escalar = leer.nextInt();
        
        // Ingresar los elementos del vector
        for (i=0; i<n; i++) {
            System.out.print("Ingrese el elemento A ["+i+"]: ");
            A[i] = leer.nextInt();
        }
        
        // Multiplicar por el escalar
        for (i=0; i<n; i++) {
            B[i] = A[i]*escalar;
        }
        
        // Presentar el vector B
        
        for(i=0; i<n; i++) {
            System.out.println("B ["+i+"] = "+B[i]);
        }
    }
}
